src/stories/Footer.stories.js

```diff
@@ -5,7 +5,7 @@ import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 import { checkA11y } from '@storybook/addon-a11y';
-import { Route, Link, MemoryRouter } from 'react-router-dom';
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
Can you tell me more about what MemoryRouter is used for in storybook? I'm taking a brief break from reading documentation :)

Reply:
I got this error `You should not use <Route> or withRouter() outside a <Router>` when I was adding a story so I googled it and MemoryRouter was the answer. 

From React Router 4 docs:
> `<MemoryRouter>` is a `<Router>` that keeps the history of your "URL" in memory (does not read or write to the address bar). Useful in tests and non-browser environments like React Native.